src/modules/custodian/dto/custodian-create-and-provision-request.dto.ts

```diff
@@ -0,0 +1,22 @@
+export interface CustodianCreateAndProvisionRequest {
+  Title: string;
+  Id?: number;
+  Code?: string;
+  TemplateId: string;
+  ParentId?: number;
+  InterestedParties?: string;
+  Secure?: boolean;
+  DoNotSubscribeInterestedParties?: boolean;
+  Links?: []; // Custodian supports sending an array of links, but we do not use this functionality.
+  FormButton?: string;
+  HasAttachments?: boolean;
+  Metadata?: MetadataItem[];
+  ConnectedListMetadata?: []; // TODO APIM-136: Mulesoft sends an empty array for this field but it is not mentioned in Custodian's documentation.
```

Comment:
@avaitonis I wanted to double-check with you, but I assume you don't have any reason why we should keep this field if it's not documented by Custodian? (https://custodian.repstor.com/swagger/ui/index)

Reply:
We can try to remove this, maybe it is legacy field.

Reply:
Thanks, I've removed it